### ISSUE TITLE
Mesh issue

### ISSUE THREAD

**Issue by WhiteCoreWebUI** [InWorldz/halcyon#432]
ok before i start this: i reverted my changes to all files about messing with caps etc and started with a complete fresh install of halcyon on a new OS however i see the same problem , mesh is either invisible or corrupted, testing with the ruth v2 avatar and several items i made in blender they either show with holes (LOD Settings to max no affect) in alchemy or inworld3 viewer, invisible on firestorm and with yellow funny triangles on Singularity , guess its my stupidness again but what could cause this? Aperture is running and also wingridproxy reports calls and answers to the getmesh cap

**Comment by kf6kjg** [comment]
Screenshots might be beneficial here.

**Comment by WhiteCoreWebUI** [comment]
![unbenannt](https://user-images.githubusercontent.com/9350749/37564975-f184b09c-2aa0-11e8-838a-eeaed4e536c1.PNG)
![unbenannt2](https://user-images.githubusercontent.com/9350749/37565004-76e1bfa0-2aa1-11e8-80db-808364072b9a.PNG)
By now i think firestorm does the same as singularity only doesnt show the triangles.

**Comment by mdickson** [comment]
Can you start with a file you know works (its been uploaded to InWorldz or SL or some other grid)? Verify thats correct first.   There are 2 code paths involved.  A Cap for uploading of assets which includes the code path to compute the Land Impact for the mesh and a seperate to actually do the upload.  If you're getting those LI numbers back then the server thinks its received a valid mesh and should store it as an asset.   The GetMesh call is redirected to Aperture by default and its used to retrieve the mesh data out of the region server.

The holes you describe often happen with meshes with too many faces.  The uploader will try and correct that by splitting the mesh but doesn't always get it right.  Could also be flipped normals.  But as I said I'd try with a known working case first.

You're using whip for asset storage?  Does the log show the asset hitting the whip server?

**Comment by WhiteCoreWebUI** [comment]
yes the ruth body i have tried in several grids opensim/sl etc only here it does not want to work at all

**Comment by WhiteCoreWebUI** [comment]
the fun part is, after the install it worked for like 4-5 logins and then started this, i assumed it was my cache that got corrupted but that was not the case after clearing it it got even worse

**Comment by mdickson** [comment]
Ok, I've made the body available in IW personally so I know that works.  You're uploading using the DAE's that shin made?  Or starting from the blend file and your own export?

**Comment by WhiteCoreWebUI** [comment]
the ones that shin made including the physics DAE box, i hear the sound rezzing, i see it in object search but it stas invisible

**Comment by emperorstarfinder** [comment]
I have seen those triangles before but they go away when that piece rezzes.  

A thought I did just have is I had forgotten to make sure the CPU for each virtual machine was at least 2 vCPUs, so if you have it set up similar to how we set up our Halcyon test grid on my team's test architecture servers, you might need to increase the number of vCPUs available to each machine.  That might be slowing down the rezzing of the content for you.

**Comment by WhiteCoreWebUI** [comment]
it is running on a 8 core machine, i figured it though it was a misconfigured nat that caused wrong packets to be send back

**Comment by emperorstarfinder** [comment]
Very cool to hear you got it sorted.  Now time for me to get back to work on my main grid.  You do already know how to get a hold of me if you need anything.  Second Galaxy does now have a lot of components from halcyon running on it along with our Halcyon test grid so feel free to test stuff there too if you are unsure if its an issue with your grid setup or not.
